packages/css/src/table.css

```diff
@@ -5,6 +5,7 @@
   --dsc-table-border-color: var(--ds-color-neutral-border-subtle);
   --dsc-table-border-divider-width: 2px;
   --dsc-table-border-divider-style: solid;
+  --dsc-table-border-divider-color: var(--dsc-table-border-color);
```

Comment:
```suggestion
  --dsc-table-divider-border-color: var(--dsc-table-border-color);
```
`border-color` is the css property

Reply:
good point, just followed what was already there, but updated now :)

Reply:
yeah, we missed this earlier, good to get it cleaned up now :)